libraries/Microsoft.Bot.Builder.AI.LUIS/LuisApplication.cs

```diff
@@ -62,7 +62,7 @@ private LuisApplication(ValueTuple<string, string, string> props)
 
             if (string.IsNullOrWhiteSpace(endpoint))
             {
-                throw new ArgumentException($"\"{endpoint}\" is not a valid LUIS endpoint.");
+                endpoint = "https://westus.api.cognitive.microsoft.com";
```

Comment:
To achieve parity with JS I think the endpoint validation should be removed from this file, and the endpoint should be assigned if null or empty in the LuisRecognizer construction. Let me know your thoughts.

Thanks

Reply:
Yeah, agree with Emilio - primarily because it's a get/set on the endpoint, and someone could sneak around the validation for fun, after initial creation..

---
In reply to: [270952889](https://github.com/Microsoft/botbuilder-dotnet/pull/1574#discussion_r270952889) [](ancestors = 270952889)